tests/health_checks/NDv4/test-ib.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+ib=200;
+num_r=0
+num_s=0
+num_ps=0
+
+#Check ibstat Rates and make sure they are all 200.
+for i in $(ibstat | grep "Rate:" | cut -d: -f2 | xargs); do 
+	num_r=$((num_r+1));
+	if [ $i -lt $ib ];
```

Comment:
Let's put `then` in this line, to be consistent with other places.

Reply:
Thanks, that was a mistake!